pkg/tenant/tenant.go

```diff
@@ -116,6 +119,13 @@ func (s *Server) Setup() (err error) {
 			// Could not connect to Quarterdeck with the specified timeout, cannot start Tenant.
 			return err
 		}
+
+		// Start the metatopic subscriber
+		s.topics = NewTopicSubscriber(s.ensign)
+		wg := &sync.WaitGroup{}
```

Comment:
Not `s.wg`?

Reply:
Oops, yeah this should definitely be `s.wg`.